test/algorithms/intersects/intersects_sph_geo.cpp

```diff
@@ -79,3 +79,29 @@ int test_main( int , char* [] )
 
     return 0;
 }
+
+//https://github.com/boostorg/geometry/issues/480
+namespace spherical {
+  using point = boost::geometry::model::d2::point_xy<double, boost::geometry::cs::spherical_equatorial<boost::geometry::degree> >;
+  using linestring = boost::geometry::model::linestring<point>;
+  using polygon = boost::geometry::model::polygon<point>;
+  using multipolygon = boost::geometry::model::multi_polygon<polygon>;
+  using mpoint = boost::geometry::model::multi_point<point>;
+}
+
+BOOST_AUTO_TEST_CASE(usaIntersectBigTile90) {
+  auto bigTileWKt     = "POLYGON((-182.25 -2.25,-182.25 85,-90 85,4.5 85,4.5 -2.25,-90 -2.25,-182.25 -2.25))";
+  auto bigTileWKtOn90 = "POLYGON((-182.25 -2.25,-182.25 90,-90 90,4.5 90,4.5 -2.25,-90 -2.25,-182.25 -2.25))";
```

Comment:
I think we never use this camelCase convention...
Also (minor), can be const

Reply:
Thanks you are right, I was just copy-pasting here from the original issue. Fixed now.